src/java/htsjdk/samtools/Defaults.java

```diff
@@ -19,9 +19,18 @@
     /** Should MD5 files be created when writing out SAM and BAM files?  Default = false. */
     public static final boolean CREATE_MD5;
 
-    /** Should asynchronous I/O be used when writing out SAM and BAM files (one thread per file).  Default = false. */
+    /** Should asynchronous I/O be used when writing out SAM, BAM, FASTQ files (one thread per file).  Default = false. */
+    @Deprecated
     public static final boolean USE_ASYNC_IO;
 
+    /** Should asynchronous I/O be used when writing out SAM and BAM files (one thread per file).  Default = false. */
```

Comment:
And fastq? Should this just say something like "Should asynchronous I/O be used by the samtools package"?